src/components/Tabs.js

```diff
@@ -1,235 +1,33 @@
 const React = require('react');
 const PropTypes = require('prop-types');
-const Radium = require('radium');
 const _pick = require('lodash/pick');
 
-const ButtonGroup = require('./ButtonGroup');
-const StyleConstants = require('../constants/Style');
+const PillTabs = require('./tabs/PillTabs');
+const StandardTabs = require('./tabs/StandardTabs');
 
-const TabsPropTypes = {
-  activeTabStyles: PropTypes.object,
-  brandColor: PropTypes.string,
-  onTabSelect: PropTypes.func.isRequired,
-  selectedTab: PropTypes.number,
-  tabs: PropTypes.array.isRequired
+const TabsTypes = {
+  standard: StandardTabs,
```

Comment:
I'd rather keep these lower case because it matches how we use the `type` prop in other components, e.g. `Button`.